pkg/appfile/dryrun/testdata/diff-input-app-with-externals.yaml

```diff
@@ -25,5 +25,6 @@ spec:
       properties:
         objects:
           - resource: deployment
+            group: apps
```

Comment:
I've spent some time thinking about this issue as I didn't want introduce a breaking change in the API.

The `shortcutExpander` restmapper looks promising but I'm not sure how to integrate it with kubevela.

Reply:
ok, i'll try that